src/lib/settings/settings.py

```diff
@@ -187,30 +219,59 @@ def get_recipients(
 
     @staticmethod
     def _read_settings(base_path: str, read_file_func, *file_names):
-        return [
-            read_file_func(os.path.join(base_path, file_name))
-            for file_name in file_names
-        ]
+        settings = []
+        for file_name in file_names:
+            try:
+                file_content = read_file_func(os.path.join(base_path, file_name))
+                settings.append(file_content)
+            except Exception as e:
+                if file_name == SettingFileNames.REPLACEMENTS:
```

Comment:
What if any other important exception occurs during replacements processing? Wouldn't that make troubleshooting process more confusing as we are silenting all exceptions during read_file_func execution?

Reply:
Good remark! Changed to skip only FileNotFound error for replacements.json